### ISSUE TITLE
After putting the name of person in search box we should not send enter, rather click on correct person 

### ISSUE THREAD

**Issue by aahnik** [aahnik/wappdriver#10]
Person to be identified by setting @ title = name

Solve This to solve issue #6

**Comment by aahnik** [comment]
implemented , see v 0.1.2 or above

**Comment by aahnik** [comment]
```python
# we will send the name to the input key box
        search_box.send_keys(name)

        try:
            person = WebDriverWait(self.driver, self.timeout).until(expCond.presence_of_element_located(
                (By.XPATH, f'//*[@title="{name}"]')))
            person.click()
            return True
```

not dynamically updating path
as it is rare to change (hope so as generic)
and high security matter.. messing up could lead to problem
